src/praisonai-agents/praisonaiagents/memory/search.py

```
@@ -14,6 +14,9 @@
 class SearchMixin:
     """Mixin class containing search and retrieval methods for the Memory class."""
     
+    # Allowed table names for SQLite queries to prevent SQL injection
+    _ALLOWED_SQLITE_TABLES = {"short_term", "long_term"}
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)